deployment-aws/terraform/loadbalancer.tf

```diff
@@ -6,7 +6,6 @@ resource "aws_lb" "api" {
   security_groups = [aws_security_group.load_balancer.id]
   access_logs {
     bucket  = "webmev-logs"
-    prefix  = "${local.stack}/loadbalancer"
```

Comment:
Should we leave just `loadbalancer` so that we can quickly navigate to specific resource logs?

Reply:
The default key prefix for AWS service logs is of the form: `AWSLogs/<acct-id>/<service>/<region>/...` and is fixed, so removing an extra prefix layer makes them quicker to access.